src/Microsoft.AspNetCore.Blazor.Server/MonoDebugProxy/ws-proxy/Startup.cs

```diff
@@ -0,0 +1,72 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Routing;
+using System.Net.WebSockets;
+using System.Threading;
+using System.IO;
+using System.Text;
+
+namespace WsProxy
+{
+
+	internal class Startup {
```

Comment:
I changed all the ws-proxy types to be `internal` instead of `public`.

Reply:
OK. I'm largely just ignoring these for now.